src/config.ts

```diff
@@ -53,6 +53,19 @@ export type ProviderConfig = {
     /* An optional function to generate the bucket asset key. */
     generateAssetKey?: (filePathOrURL: string, folder: string) => string;
   };
+
+  'cloudflare-r2'?: {
+    endpoint: string;
+    accountId: string;
```

Comment:
remove accountId too, it's part of endpoint as well. we can extract it from the endpoint if needed.

Reply:
👍 changed it. Will now take it from the endpoint url